app/controllers/users_controller.rb

```diff
@@ -121,6 +121,17 @@ def download_photos
     redirect_to current_user.exported_photos_file.url
   end
 
+  def import_profile
+    if params[:user]==nil
```

Comment:
- Layout/SpaceAroundOperators: Surrounding space missing for operator `==`.
- Style/NilComparison: Prefer the use of the `nil?` predicate.